lua/dotfyle_metadata/keymaps.lua

```diff
@@ -1,32 +1,51 @@
 ---Get all user defined keymaps
 ---@return table
+
+local function check_map(keymap)
+  -- check if the mapping is one we want to include
+
+  if  keymap.mode ~= "n" and keymap.mode ~= "i"
+  and keymap.mode ~= "x" and keymap.mode ~= "v"
+  then
+    return false
+  end
+  if string.find(keymap.lhs, "<Plug>")
+  or string.find(keymap.rhs, "<Plug>")
```

Comment:
This one is valid, users can define a keymap assigned to a `<Plug>` keymap oh `rhs`. So we can omit this.